api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerClientSpec.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class KafkaMirrorMakerClientSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String bootstrapServers;
+    protected Map<String, Object> config;
+    private KafkaMirrorMakerTls tls;
+    private KafkaMirrorMakerAuthentication authentication;
+    private Map<String, Object> additionalProperties;
+
+    @Description("Authentication configuration for connecting to the source cluster.")
```

Comment:
Is this right, given this method will be inherited by both consumer and producer?